pkg/auth/auth_handler.go

```diff
@@ -92,6 +100,7 @@ func (a *AuthHandler) AuthAgentAuth(token string) (interface{}, error) {
 		a.log.Errorf("Unable to fetch user's capabilities: %v", err)
 		return nil, common.ApiErrorWithDefaultInfraError(err, http.StatusUnauthorized)
 	}
+	a.Cache.Set(token, user, cache.DefaultExpiration)
```

Comment:
`DefaultExpiration` Should be part of the configuration.  relevant to the comment in line 45

Reply:
`DefaultExpiration` is the one passed in constructor